launch.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>SMART on FHIR Starter App</title>    
+  </head>
+  Loading...
+  <body>
+    <script src='./src/js/starter_app.js'></script>
```

Comment:
Dont need to include 'starter_app.js' since its not going to be used in this html file.